cmd/tiller-proxy/internal/handler/handler.go

```diff
@@ -43,7 +43,10 @@ func returnForbiddenActions(forbiddenActions []auth.Action, w http.ResponseWrite
 
 // TillerProxy client and configuration
 type TillerProxy struct {
+	// TODO(mnelson): was DisableUserAuthCheck just an initial compatibility flag when the auth check
+	// was added? (ie. can it be removed).
```

Comment:
yes, it was also a way for people to disable the auth in case they want to skip authentication. I don't think it's being used anyway so we can remove it if we want.

Reply:
Great, I'll do that separately.